packages/blade/src/components/Input/DropdownInputTriggers/BaseDropdownInputTrigger.tsx

```diff
@@ -194,7 +194,7 @@ const _BaseDropdownInputTrigger = (
       }
 
       return getTagsGroup({
-        tags: selectedIndices.map((selectedIndex) => options[selectedIndex].title),
+        tags: selectedIndices.map((selectedIndex) => options[selectedIndex]?.title),
```

Comment:
didn't quite understand the fix? if we mark an ActionListItem as disabled then  title prop doesn't exist?

Reply:
Yeah, I think so. It's also hard to reproduce locally. @saurabhdaware you may have more context here.

Reply:
We follow similar pattern at other places too:

https://github.com/razorpay/blade/blob/de82795aab4917d04be94c2b42605ae1f280c268/packages/blade/src/components/Dropdown/dropdownUtils.ts#L322

Reply:
will try out locally and check

Reply:
Will need to check their usage - https://github.com/razorpay/blade/issues/1721#issuecomment-1770084559

I wasn't able to replicate. The bug on sandbox is some react refresh dev-server issue which gets fixed on page refresh.

Reply:
So the issue on sandbox is different from issue in actual code. They both end up failing on the same point and throw same error but reasons are different.

I think putting `?` should be fine here. It will stop the error. We can see if it works out for them. My guess is this will remove the error but still won't let them achieve what they are trying to achieve.

The issue is the options are dynamic in their case and seem to change their properties. Everytime the ActionListItem changes, the options need to be recalculated and can intermediately become unavailable.